### PR TITLE
Task #3979 Added assertions and waiter to avoid StaleElementReferenceException

### PR DESCRIPTION
**Pull request by BestNastos**

### CODE REVIEW

File: jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/asserts/navigation/DesktopStepperAssert.java [hunk 1]
```diff
@@ -62,4 +63,33 @@ public DesktopStepperAssert allStepsIncomplete() {
...
+
+    /**
+     * Waits until step with the given number is completed.
+     *
+     * @param stepNumber step number in this stepper
+     * @return this {@link DesktopStepperAssert} instance
+     */
+    @JDIAction("Assert that step '{0}' in '{name}' is completed")
+    public DesktopStepperAssert stepCompleted(int stepNumber) {
+        SoftAssert.jdiAssert(new Timer(base().getTimeout() * 1000L)
```
Comment (GrebNat) [inline jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/asserts/navigation/DesktopStepperAssert.java:75]: new Timer(base().getTimeout() * 1000L

please, move the expression to variable

File: jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/asserts/navigation/DesktopStepperAssert.java [hunk 2]
```diff
@@ -62,4 +63,33 @@ public DesktopStepperAssert allStepsIncomplete() {
...
+    /**
+     * Checks if step with the given number is incomplete.
+     *
+     * @param stepNumber step number in this stepper
+     * @return this {@link DesktopStepperAssert} instance
+     */
+    @JDIAction("Assert that step '{0}' in '{name}' is incomplete")
+    public DesktopStepperAssert stepIncomplete(int stepNumber) {
+        SoftAssert.jdiAssert(element().hasStepCompleted(stepNumber - 1) ?
+                        "has step completed" : "doesn't have step completed",
```
Comment (GrebNat) [inline jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/asserts/navigation/DesktopStepperAssert.java:91]: "has step completed" 

also duplication of the string, move it to constant, please

**Review by GrebNat — CHANGES_REQUESTED** [review]

**Comment by BestNastos** [comment]
> 

Hello, Natalia, I updated my PR. Could you please take a look?

File: jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/asserts/navigation/DesktopStepperAssert.java (hunk 2 above)
Comment (BestNastos) [reply jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/asserts/navigation/DesktopStepperAssert.java]: Done

File: jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/asserts/navigation/DesktopStepperAssert.java (hunk 1 above)
Comment (BestNastos) [reply jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/asserts/navigation/DesktopStepperAssert.java]: Done

**Pull request merged**